workers/loc.api/sync/get-risk/index.js

```diff
@@ -0,0 +1,138 @@
+'use strict'
+
```

Comment:
What i think would be simpler is to `getter` function pass the `fragment` (in case example would be daily)
And then just sum a total fragment (EUR with EUR, USD with USD, etc) and then sum all fragments as to do a total of the period.